package.json

```diff
@@ -22,7 +22,7 @@
     "build": "lerna run build",
     "clean": "lerna run clean",
     "node": "yarn contracts app:node",
-    "dev": "concurrently --restart-tries 5 --kill-others \"yarn run node\" \"yarn contracts app:setup && yarn app dev\"",
+    "dev": "concurrently --restart-tries 5 --kill-others \"yarn run node\" \"yarn contracts app:setup && yarn contracts app:load && yarn app dev\"",
```

Comment:
Why fund the accounts in a separate script instead of in the `app.ts` script? My only thought is that it might be nice to do this funding first in the same file, as it _might_ be easier to initialize with various states as things get more complex. I think this is fine though and don't feel strongly either way!

Reply:
Your `setBalance` fn is a lot faster than the other approach so I've replaced it with that! (5871fbb)

Reply:
Your setBalance fn is a lot faster than the other approach so I've replaced it with that! (5871fbb) 🙌